shrike-examples/contoso/utils/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""make basic utils importable from contoso.utils for convenience and readability"""
+from .log_utils import log, DataCategory, CompliantLoggerInitializer
+from .arg_utils import str2bool, str2intlist, commastring2list
+from .file_utils import select_first_file
+from .aml_utils import (
+    AmlRunWrapper,
+    aml_run_attach,
+    log_directory,
+    log_metric,
```

Comment:
Yes, I believe you're right. However, I'm not sure that all of these are readily available in shrike, which means some non-trivial amount of work might required. Since these are supposed to be part of the probe component black box, I don't think it was necessary to do it for these problems. If you feel strongly about this, let hop on a call and argue :)